gtk/CMakeLists.txt

```diff
@@ -188,6 +198,7 @@ endif()
 
 if(ENABLE_NLS)
     install(FILES ${${PROJECT_NAME}_DESKTOP_FILE} DESTINATION ${CMAKE_INSTALL_DATADIR}/applications)
+    install(FILES ${${PROJECT_NAME}_APPDATA_FILE} DESTINATION ${CMAKE_INSTALL_DATADIR}/appdata)
```

Comment:
Either is fine, the former is the new location which may not be available on all distros, and the old location is going to be supported forever. I use the latter in my projects FWIW.